lib/support/chrome-perflog-parser.js

```diff
@@ -140,6 +141,15 @@ module.exports = {
             log.error('Error parsing response: ' + JSON.stringify(data, null, 2));
             throw e;
           }
+
+          let previousEntry = previousConnectionEntries.get(data.response.connectionId);
+          if(previousEntry) {
+            let startedDateTime = moment(entry.startedDateTime).valueOf();
```

Comment:
Avoid converting to and from string representation of timestamp here. Instead use entry._wallTime